pytest_fixtures/core/broker.py

```diff
@@ -96,9 +96,20 @@ def module_discovery_sat(
     discovery_org.value = module_sca_manifest_org.name
     discovery_org.update(['value'])
 
+    # Verify the settings are updated
+    assert (
+        sat.api.Setting().search(query={'search': 'name=discovery_location'})[0].value
+        == module_location.name
+    )
+    assert (
+        sat.api.Setting().search(query={'search': 'name=discovery_organization'})[0].value
+        == module_sca_manifest_org.name
+    )
+
     # Enable flag to auto provision discovered hosts via discovery rules
     discovery_auto = sat.api.Setting().search(query={'search': 'name=discovery_auto'})[0]
     discovery_auto.value = 'true'
     discovery_auto.update(['value'])
+    assert sat.execute('cat /dev/null > /var/lib/dhcpd/dhcpd.leases').status == 0
```

Comment:
Do we really need this workaround now?

Reply:
No. we do not see the issue now of unable to discover the host. I will remove it.